.circleci/config.yml

```diff
@@ -155,7 +166,7 @@ jobs:
             export STITCH_CONFIG_DIR=/tmp/${CIRCLE_PROJECT_REPONAME}
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
             pip install pymongo==4.4.0
-            selected_tests=$(circleci tests glob "tests/*{data,interrupt,open,restrict,view}*")
+            selected_tests=$(circleci tests glob "tests/*{test_mongodb}*")
```

Comment:
[According to the docs](https://circleci.com/docs/use-the-circleci-cli-to-split-tests/#glob-test-files), it seems like this can be simplified to just `*test_mongodb*`. Which is also redundant now since all of our tests start with `test_mongodb`. So `test_mongodb*` should work

Reply:
Okay, got it. Will make this change in latest commit.